<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AkonadiContactActions</class>
 <widget class="QWidget" name="AkonadiContactActions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>408</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Show Address</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="KComboBox" name="ShowAddressAction"/>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Url:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KLineEdit" name="kcfg_AddressUrl"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Command:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KLineEdit" name="kcfg_AddressCommand"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_8"/>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>9</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Dial Phone Number</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="KComboBox" name="DialPhoneNumberAction"/>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget_2">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="page_3"/>
        <widget class="QWidget" name="page_4">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Command:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KLineEdit" name="kcfg_PhoneCommand"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_7"/>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Send SMS</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="KComboBox" name="SendSmsAction"/>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget_3">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_5"/>
        <widget class="QWidget" name="page_6">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Command:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KLineEdit" name="kcfg_SmsCommand"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
